app/csv_generator/pacbio_sample_sheet.rb

```diff
@@ -30,7 +30,7 @@ def generate
   private
 
   def wells
-    run.plates.first.wells
+    run.plates.collect(&:wells).flatten
```

Comment:
You might like the built-in `flat_map` function from Ruby -> https://apidock.com/ruby/Enumerable/flat_map